brion/circuit.h

```diff
@@ -26,8 +26,6 @@
 namespace brion
 {
 
-namespace detail { class Circuit; }
-
 /** Read access to a Circuit file.
  *
  * Following RAII, this class is ready to use after the creation and will ensure
```

Comment:
this RAII keyword information is really needed I wonder. I never saw in another API doc, or thats only me.